<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>183</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>QtSpotify</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0" colspan="4">
     <widget class="QLabel" name="albumLabel">
      <property name="font">
       <font>
        <family>Victor Mono</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QLabel" name="songLabel">
      <property name="font">
       <font>
        <family>Victor Mono</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="4">
     <widget class="QLabel" name="artistLabel">
      <property name="font">
       <font>
        <family>Victor Mono</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="4">
     <widget class="QProgressBar" name="progressBar">
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="3" rowspan="2">
     <widget class="QPushButton" name="nextButton">
      <property name="font">
       <font>
        <family>Victor Mono</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2" rowspan="2">
     <widget class="QPushButton" name="playButton">
      <property name="font">
       <font>
        <family>Victor Mono</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1" rowspan="2">
     <widget class="QPushButton" name="pauseButton">
      <property name="font">
       <font>
        <family>Victor Mono</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Pause</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" rowspan="2">
     <widget class="QPushButton" name="previousButton">
      <property name="font">
       <font>
        <family>Victor Mono</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Previous</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4" rowspan="6">
     <widget class="QSlider" name="volumeSlider">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
